docs/ref/models/instances.txt

```diff
@@ -589,6 +589,24 @@ row. In these cases you can pass the ``force_insert=True`` or
 Passing both parameters is an error: you cannot both insert *and* update at the
 same time!
 
+.. versionchanged:: 5.0
+
+    In case of an inherited model the ``force_insert=True`` only forces the
+    insertion in the leaf model. ``force_insert=False`` is taken as default for the
+    parent models.
+
+    If it is required to force insert the parent models also, then consider passing
+    a tuple of Model classes that requires to perform force insertion. For
+    example::
+
+        Child(id=3, name="Child-3", value=10).save(force_insert=(GrandParent, Parent))
+
+    ``force_insert=(models.Model,)`` can be used in order to perform forced
+    insertion of all the base classes. For example::
+
+        Child(id=1, name="Child-1", value=10).save(force_insert=(models.Model,))
```

Comment:
Any thoughts about allowing passing a single model class instead of a singleton (e.g. `GrandParent` instead of `(GrandParent,)`).

I seems that it would be coherent with the signature of `issubclass` which allows either a `type` or `tuple[type]` ot be provided. Passing a tuple is also only useful for MTI models that are definitive multiple parents at once which is atypical scenario.